web/src/recent_view_data.ts

```diff
@@ -7,73 +7,76 @@ export type ConversationData = {
     participated: boolean;
     type: "private" | "stream";
 };
-export const conversations = new Map<string, ConversationData>();
```

Comment:
I recommend hiding whitespace to view this diff

<img width="298" alt="image" src="https://github.com/zulip/zulip/assets/5634097/cc1033b1-a171-4495-bef7-97df47b5b83e">

Reply:
Yeah I use that all the time for refactors.